packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/__snapshots__/opsGraph.test.js.snap

```diff
@@ -20,23 +20,13 @@ exports[`<OperationsGraph> Graph rendered successfully 1`] = `
 <div
   className="ops-container"
 >
-  <XYPlot
-    className=""
+  <ForwardRef
```

Comment:
ideally we should get rid of snapshot testing here, such large snapshot is useless and highly dependent on implementation details of the lib